src/react/components/common/assetPreview/videoAsset.tsx

```diff
@@ -137,6 +150,16 @@ export class VideoAsset extends React.Component<IVideoAssetProps> {
         }
     }
 
+    private movePreviousFrame = () => {
```

Comment:
Question about whether these key-frames should be normalized from the beginning of the video vs. arbitrary from the current time in the video... 

Ex) If you manually scrub and pause the video, the frame rate will now be offset vs. if you manually progressed through the keyframes.

Reply:
It should probably snap to nearest keyframe. In my other version I had it normalized. I can add that back in since it got removed